packages/moonstone/Switch/Switch.module.less

```diff
@@ -101,13 +118,16 @@
 			}
 		});
 	}
-}, parent);
 
-// When a parent is disabled
-.disabled({
-	.toggleIcon {
-		.applySkins({
-			opacity: @moon-disabled-opacity;
-		});
-	}
-});
+	.disabled({
+		.toggleIcon {
+			.applySkins({
+				opacity: @moon-disabled-opacity;
+
+				// .disabled({
```

Comment:
Is this needed?

Reply:
"needed" may be an overstatement. It can be safely removed.